release-github/action.yml

```
@@ -46,4 +64,5 @@ runs:
           ${{ inputs.library-name }}/*.whl
           ${{ inputs.library-name }}/*.tar.gz
           documentation-html.zip
-          documentation-pdf/*.pdf
+          documentation-pdf.zip
+          other-artifacts/**/*-wheelhouse-*.whl
```

I think this should still be ZIP... not whl
```suggestion
          other-artifacts/**/*-wheelhouse-*.zip
```

---

I thought artifacts were decompressed when downloaded inside of the actions.

---

Ops, sorry. It looks like the wheelhouse already ships as a ZIP. Makes sense, it is a big collection of wheels...